types/hls.js/index.d.ts

```diff
@@ -189,33 +190,61 @@ declare namespace Hls {
      */
     interface Level {
         /**
-         * level url. might contain sever items if failover/redundant streams are found in the manifest
+         * attribute list
          */
-        url: string[];
+        attrs: Array<{[key: string]: string}>;
```

Comment:
Personally I prefer `{[key: string]: string}[]` or even better to define an interface or a type
`interface AttrList { [key: string]: string }`

Reply:
I will use your second approach then. The first one raises a linter error.